fieldsolver/gridGlue.cpp

```diff
@@ -19,33 +19,33 @@ int getNumberOfCellsOnMaxRefLvl(dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geomet
 
    int nCells = 0;
    auto maxRefLvl = mpiGrid.mapping.get_maximum_refinement_level();
-   
+
    for (auto cellid : cells) {
       auto refLvl = mpiGrid.get_refinement_level(cellid);
       nCells += pow(pow(2,maxRefLvl-refLvl),3);
    }
 
    return nCells;
-   
+
 }
 
 
-/*Compute coupling DCCRG <=> FSGRID 
+/*Compute coupling DCCRG <=> FSGRID
 
   onDccrgMapRemoteProcess   maps fsgrid processes (key) => set of dccrg cellIDs owned by current rank that map to  the fsgrid cells owned by fsgrid process (val)
 
-  onFsgridMapRemoteProcess  maps dccrg processes  (key) => set of dccrg cellIDs owned by dccrg-process that map to current rank fsgrid cells 
+  onFsgridMapRemoteProcess  maps dccrg processes  (key) => set of dccrg cellIDs owned by dccrg-process that map to current rank fsgrid cells
   onFsgridMapCells          maps remote dccrg CellIDs to local fsgrid cells
 */
 
 template <typename T, int stencil> void computeCoupling(dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>& mpiGrid,
```

Comment:
So this is a legit swap of tabs to spaces, but I don't see a systematic fixing of where such argument lists should be indented. Any cure for that?